test/helpers/asl-db.js

```diff
@@ -6,6 +6,7 @@ module.exports = settings => {
     init: (populate, keepAlive) => {
       const schema = Schema(settings);
       const tables = [
+        'AsruEstablishment',
```

Comment:
Not 100% sure what state I got in to require this, but my test run blew up on 100% of tests without it.